src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -900,6 +940,18 @@ private IOperation CreateBoundConversionOperation(BoundConversion boundConversio
             BoundExpression boundOperand = boundConversion.Operand;
             if (boundConversion.ConversionKind == CSharp.ConversionKind.MethodGroup)
             {
+                if (boundConversion is { Type: FunctionPointerTypeSymbol { Signature: var signature }, Syntax: var expression })
```

Comment:
>{ Signature: var signature }, Syntax: var expression } [](start = 73, length = 54)

Are these variable declarations needed? `signature` is unused, and the `boundConversion.Syntax` can be used instead of `expression` (in the same way that `boundConversion.Operand` and `boundConversion.SymbolOpt` are).